host-bmc/dbus/custom_dbus.cpp

```diff
@@ -92,8 +92,9 @@ void CustomDBus::implementCableInterface(const std::string& path)
     }
 }
 
-void CustomDBus::updateItemPresentStatus(const std::string& path,
-                                         bool isPresent)
+void CustomDBus::updateInventoryItemProperties(const std::string& path,
+                                               bool isPresent,
+                                               const std::string& prettyName)
```

Comment:
make this an std::optional<std::string>& instead

Reply:
Done